Strings/src/com/company/task_2/Task_1.java

```diff
@@ -0,0 +1,33 @@
+package com.company.task_2;
+
+public class Task_1 {
+    public static void main(String[] args) {
+        String shakespeareLine = "    Meet my family. There are five of us – my parents, my elder brother,\n"
+                + "my baby    sister and me. First,   meet my mum and dad, Jane and Michael. \n"
+                + "My mum enjoys reading and my dad enjoys playing chess with my brother Ken. \n"
+                + "My mum is slim and rather tall. She has long red hair and big brown eyes. \n"
+                + "She has a very    pleasant smile and a soft voice. My mother is very kind and understanding.   \n"
+                + "We are real friends. She is a housewife. As she has three children, she is always busy around the house. \n"
+                + "She takes care of my baby sister Meg, who is only three months old.    ";
+
+        System.out.println("String:\n" + shakespeareLine);
+        System.out.println("String with changes:\n" + findMoreSpaces(shakespeareLine));
+    }
+
+    private static int findMoreSpaces(String line) {
+        int counter = 0;
+        int maxCount = 0;
+
+        for (int i = 0; i < line.length(); i++) {
+            if (line.charAt(i) == ' ') {
```

Comment:
there is a practice to use in such cases static variable instead of just literal just for better readability
`private static final char SPACE = ' ';`

Reply:
Исправил